components/Button/Button.module.css

```diff
@@ -0,0 +1,45 @@
+.button {
+  font-family: 'Jost';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 13px;
+  line-height: 19px;
+  width: 80px;
+  height: 30px;
+  background: #AD1FEA;
+  color: #F2F4FE;
+  border-radius: 10px;
+}
+
+.button:hover {
+  background: #C75AF6;
+}
+
+/* Variants */
+
+/* Close */
+.button_close {
+  background: #3A4374;
+}
+
+.button_close:hover {
+  background: #656EA3;
+}
+
+/* Delete */
+.button_delete {
```

Comment:
I like the approach with variants, but I don't like the naming. 
I think the delete variant is red, but a red button could be used for other purposes somewhere else in the app. You might use a red button for cancel. 

Likewise, the button_close variant is grey which could be used for other purposes: going back, clearing a form etc etc. 

A common strategy is to abstract these names into a range of meanings for instance. This seperates it from a specific usage like close or delete and from a specific color like red or green (colors can change with themes and changes in design) 

Examples:

- danger -> a button that can be used to delete, cancel or other sensitive action, typically red
- success -> a button that can be used to create something new, or confirm a success message, typically green

Here is bootstraps naming scheme

<img width="907" alt="Screenshot 2023-01-04 at 13 02 52" src="https://user-images.githubusercontent.com/20372832/210551384-b6e057ff-f1cb-4702-9f34-289f2465abcb.png">